### ISSUE TITLE
QR Code missing Logo when downloaded as SVG

### ISSUE THREAD

**Issue by steven-tey** [dubinc/dub#1083]
When downloaded as SVG:

![qrcode](https://github.com/user-attachments/assets/26ad561f-50aa-44fd-aa0e-d2a82c292861)

Mac preview:

![CleanShot 2024-07-17 at 15 19 24](https://github.com/user-attachments/assets/b86bf2fa-950b-4a5f-8d1f-7bb0bd4a8ee8)

Interestingly, PNG & JPEG works fine:

![qrcode](https://github.com/user-attachments/assets/f0b65f74-96ee-464d-9ce4-db07c63a6376)

**Comment by taskane** [comment]
Can you ellaborate the issue? How could i recreate it

**Comment by rahulsingh3526** [comment]
can you assign this task to me ?

**Comment by goyalaayush** [comment]
SVG On Windows:
![Screenshot 2024-07-20 231624](https://github.com/user-attachments/assets/083bdbe8-222d-408b-9945-0c2a4da6895c)

Can you tell me how I can recreate this issue?

**Comment by mohammedyunus2002** [comment]
![Screenshot 2024-07-21 at 12 16 58 AM](https://github.com/user-attachments/assets/eeb9b732-39be-43be-b4b9-df2f93510d11)

It works fine for mac as well

**Comment by steven-tey** [comment]
@goyalaayush @mohammedyunus2002 yeah this one is super interesting – looks like it works when you open the SVG in the browser, but it breaks on Mac Preview:

![CleanShot 2024-07-22 at 09 22 50](https://github.com/user-attachments/assets/99e9eac4-0acb-4ae7-8b9b-b051b6f8e14a)

![CleanShot 2024-07-22 at 09 23 29](https://github.com/user-attachments/assets/276052a3-0b94-4b99-b18d-c3176819dba4)

It also shows up with an empty logo on Figma:

![CleanShot 2024-07-22 at 20 57 46](https://github.com/user-attachments/assets/43acea36-ab0a-453d-a1e8-b01a4fcf040c)

**Comment by steven-tey** [comment]
Good news – this has been fixed, thanks to @devkiran 🐐 

https://github.com/dubinc/dub/pull/1101
